frontend/src/config/mozambique/prism.json

```diff
@@ -503,6 +503,7 @@
     },
     {
       "title": "Summer Dashboard",
```

Comment:
Should we simply use a kebab-case version of the title? wdyt?

Reply:
I agree that having a path makes sense right now, but my worry is that we're in an in-between state :
- it's not super useful if we only have preconfigured dashboards that have a somewhat fixed config.title (regardless of what users end up changing it to)
- if we end up moving towards a more complicated approach where users can create their own dashboards and publish them, paths would become obsolote and we would likely have to move to unique ids anyway.

I'm fine either way, I just feel like it's a not super needed complexity given our current "in-between" state :)